### DISCUSSION TITLE
Spinner Not Showing on Opened Slider Image in Fancybox 5

### DISCUSSION THREAD

**Discussion by Everett-Young** [fancyapps/ui#687] [Q&A] [answered]
Hi everyone,

I'm encountering an issue with Fancybox 5 where I try to add a loading spinner to an open image in a slider using the showLoading method. Instead of displaying the spinner, the screen goes black with no spinner visible.

Has anyone faced this problem before, or does anyone have any ideas on why this might be happening?

Thanks in advance for your help!

**Comment by fancyapps** [comment, marked as answer]
Hi,

Sorry, there seems to be an issue with that method. As a workaround, you can do something like this:

```js
Fancybox.getSlide().spinnerEl = null;
Fancybox.getInstance().showLoading(Fancybox.getSlide());
```

or simply add your own loading icon directly to the slide element (e.g., `Fancybox.getSlide().el`)

**Reply by Everett-Young** [reply]
Thank you for your suggestion! I tried it:
`Fancybox.getSlide().spinnerEl = null;`
, but now hideLoading isn't working properly—the spinner stays visible even after it's supposed to be hidden. Any ideas on how to fix this?

**Reply by fancyapps** [reply]
Well, it seems to work - https://jsfiddle.net/aymq39j5/ - can you create a demo where that issue could be inspected?

**Reply by Everett-Young** [reply]
Hmm, you're right, it does work. I must have missed something during debugging. Thank you for your help!
